_i18n/en.yml

```diff
@@ -155,7 +155,7 @@ developers_page:
   footer:
     heading: Contact us
     p_1: If you’re interested in learning more about the platform, please [fill out this form](https://share.hsforms.com/16DIoo--rTU2xbNW1MShkBg3ak9e){:target="_blank"}.
-  intro:
+    intro:
```

Comment:
Thanks for the catch @zachmargolis

Reply:
Sure! Looks like the indentation was still messed up a bit so I just pushed a commit to fix

Reply:
👍 thanks!